src/plugins/stt/deepgram/DeepgramSpeechRecognition.ts

```diff
@@ -0,0 +1,47 @@
+import { ListenLiveClient, LiveSchema, createClient } from '@deepgram/sdk';
+import { LiveTranscriptionEvents } from '@deepgram/sdk';
+
+export class DeepgramSpeechRecognition {
+  live: ListenLiveClient | undefined;
+  mediaRecorder: MediaRecorder | undefined;
+
+  start(
+    apiKey: string,
+    onSpeechRecognized: (transcript: string) => void,
+    deepgramOptions?: LiveSchema,
+  ) {
+    navigator.mediaDevices.getUserMedia({ audio: true }).then((stream) => {
+      this.mediaRecorder = new MediaRecorder(stream);
+
+      const deepgram = createClient(apiKey);
+      this.live = deepgram.listen.live({
+        model: 'nova-2',
+        language: 'en-US',
+        ...deepgramOptions,
+      });
+
+      this.live.on(LiveTranscriptionEvents.Open, () => {
+        console.log('started speech recognition');
```

Comment:
suggestion: remove all logs from our sdk (until we have a proper logging that is only enabled if the user wants to)

Reply:
Done: https://github.com/agencyenterprise/alpha-ai-avatar-sdk-react/pull/7/commits/f332ea1976b8b8928986085970af107313ded53e